addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -284,6 +300,7 @@ def _get_move_dict_vals_change_period(self):
                 }),
             ]
 
+        reference_move.invalidate_recordset()
```

Comment:
Uh? What? Why?

Reply:
Yes, I'm ok for using `new()`, but why do you need to invalidate?

Reply:
But is the `invalidate_recordset` necessary? I don't know actually... :wink:
I'd appreciate it if you can explain hahaha